tests/_support/loadGrunt.ts

```diff
@@ -0,0 +1,79 @@
+import * as grunt from 'grunt';
+import * as path from 'path';
+import * as mockery from 'mockery';
+import * as _ from 'lodash';
+import { IRootRequire } from 'dojo/loader';
+
+declare const require: IRootRequire;
+
+export interface MockList {
+	[key: string]: any;
+}
+
+export interface TaskLoadingOptions {
+	peerDependencies?: {
+		[key: string]: any;
+	};
+}
+
+export function runGruntTask(taskName: string, callback?: () => void) {
```

Comment:
we talked on ScreenHero about this, but this doesn't apply to our multitasks. We can keep it; it doesn't help us ATM.